Lombiq.Hosting.Azure.ApplicationInsights/Extensions/ApplicationInsightsInitializerExtensions.cs

```diff
@@ -23,6 +24,13 @@ public static class ApplicationInsightsInitializerExtensions
         IConfiguration configurationManager)
     {
         var services = builder.ApplicationServices;
+
+        services.Configure<TelemetryConfiguration>(config =>
+        {
+            var credential = new ManagedIdentityCredential();
```

Comment:
We also need local authentication for local development, and the Readme extended with how you'd set that up. I.e., I'm a developer, adding this module to my app (or running it as part of OSOCE), what should I do to make the quick pulse telemetry channel work locally? (I'll also link to this from the Confluence wiki of the project.)

Conversely, what should I do (differently) when deploying it to staging/prod?

Reply:
So, please document this. This isn't enough:

> To set up Entra Authentication follow the steps that most closely match your situation over at [Microsoft Entra authentication for Application Insights](https://learn.microsoft.com/en-us/azure/azure-monitor/app/azure-ad-authentication?tabs=net)

One needs to dig into the Microsoft documentation to figure out where to start. Add docs pointing people to specifics: one scenario is running this locally, and the other is running it in an Azure service. The second case can be anything but if this matters, point to the most common use cases of Azure App Services and Container Instances.

Reply:
I've updated it to be more straightforward and also explain the setup for local development. Let me know if more changes are needed